style.css

```diff
@@ -2,579 +2,1338 @@
+    width: 131px;
+    height: 48px;
+    margin: 24px 122px 0 122px;
+    background: #ff6b00;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    color: #ffff;
+    border: hidden;
+  }
 
-.contactform hr {
-  width: 375px;
-  height: 1px;
-  margin-top: 124px;
-}
+  .contactform hr {
+    width: 375px;
+    height: 1px;
+    margin-top: 124px;
+  }
 
-.contact-social-icons {
-  width: 200px;
-  margin: 15px 0 0 70px;
-  padding: 0;
-}
+  .contact-social-icons {
+    width: 200px;
+    margin: 15px 0 0 70px;
+    padding: 0;
+  }
 
-.contact-social-icons ul {
-  list-style: none;
-  display: flex;
-  padding: 0;
-  margin: 0;
-}
+  .contact-social-icons ul {
+    list-style: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
 
-.contact-social-icons li {
-  align-items: center;
-  align-self: center;
-  margin-left: 20px;
-}
+  .contact-social-icons li {
+    align-items: center;
+    align-self: center;
+    margin-left: 20px;
+  }
 
-.contact-social-icons img {
-  width: 20px;
-  height: auto;
+  .contact-social-icons img {
+    width: 20px;
+    height: auto;
+  }
+
+  .contact-social-icons hr {
+    width: 134px;
+    height: 5px;
+    margin: 23px 0 8px 50px;
+    background: #0d151e;
+    border-radius: 10px;
+  }
 }
 
-.contact-social-icons hr {
-  width: 134px;
-  height: 5px;
-  margin: 23px 0 8px 50px;
-  background: #0d151e;
-  border-radius: 10px;
+@media (min-width: 760px) {
+  * {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+
+  .moble-container {
+    display: none;
+  }
+
+  .desktop-nav-container {
+    display: flex;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    justify-content: flex-end;
+    align-items: center;
+    position: fixed;
+    overflow: hidden;
+    background-color: #3e3c3c;
+  }
+
+  .desklogo {
+    width: 120px;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    cursor: pointer;
+    text-decoration: none;
+    margin-right: auto;
+    margin-left: 100px;
+  }
+
+  .nav-links a {
+    transition: all 0.3s ease 0s;
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+  }
+
+  .nav-links {
+    display: flex;
+    list-style: none;
+    padding: 0;
+  }
+
+  .nav-links li {
+    padding: 26px 0 26px 32px;
+  }
+
+  .nav-links img {
+    margin-right: 50px;
+  }
+
+  .desktop-headline {
+    display: flex;
+    min-height: 870px;
+    background-image: url(images/Illustration-header-desk.png);
+    background-color: #1c1a19;
+    background-repeat: no-repeat;
+    background-size: 30% auto;
+    background-position: right;
+  }
```

Comment:
Kindly ensure that your design matches the actual figma design at all screen widths of the design. The image has pushed into the text. There is also a big white empty space to the right side of the design which is not suppose to be there. Kindly check and fix them. 

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-12 12-17-57](https://user-images.githubusercontent.com/65233860/201474114-9fbd2b5c-7b7b-47bd-aa5b-9a5ce877a393.png)  |  ![Screenshot from 2022-11-12 12-19-40](https://user-images.githubusercontent.com/65233860/201474125-98bdeb83-214f-4412-aa24-3ca6c97fb079.png)